docs/docs/frigate/hardware.md

```diff
@@ -33,17 +33,40 @@ My current favorite is the Minisforum GK41 because of the dual NICs that allow y
 
-## Google Coral TPU
+## Detectors
 
-It is strongly recommended to use a Google Coral. Frigate is designed around the expectation that a Coral is used to achieve very low inference speeds. Offloading TensorFlow to the Google Coral is an order of magnitude faster and will reduce your CPU load dramatically. A $60 device will outperform $2000 CPU. Frigate should work with any supported Coral device from https://coral.ai
+A detector is a device which is optimized for running inferences efficiently to detect objects. Using a recommended detector means there will be less latency between detections and more detections can be run per second. Frigate is designed around the epectation that a detector is used to achieve very low inference speeds. Offloading TensorFlow to a detector is an order of magnitude faster and will reduce your CPU load dramatically. As of 0.12, Frigate supports a handful of different detector types with varying inference speeds and performance.
+
+### Google Coral TPU
+
+It is strongly recommended to use a Google Coral. A $60 device will outperform $2000 CPU. Frigate should work with any supported Coral device from https://coral.ai
 
 The USB version is compatible with the widest variety of hardware and does not require a driver on the host machine. However, it does lack the automatic throttling features of the other versions.
 
 The PCIe and M.2 versions require installation of a driver on the host. Follow the instructions for your version from https://coral.ai
 
 A single Coral can handle many cameras and will be sufficient for the majority of users. You can calculate the maximum performance of your Coral based on the inference speed reported by Frigate. With an inference speed of 10, your Coral will top out at `1000/10=100`, or 100 frames per second. If your detection fps is regularly getting close to that, you should first consider tuning motion masks. If those are already properly configured, a second Coral may be needed.
 
-### What does Frigate use the CPU for and what does it use the Coral for? (ELI5 Version)
+### OpenVino
+
+The OpenVINO detector type is able to run on x86 hosts that have an Intel CPU, GPU, or VPU Hardware (ex: Intel NCS2). Arm64 hosts are also supported with VPU hardware.
+
+Inference speeds vary greatly depending on the CPU, GPU, or VPU used, some known examples are below:
+
+| Name                | Inference Speed | Notes                                                                 |
+| Intel Celeron J4105 | ~ 25 ms         | Inference speeds on CPU were ~ 150 ms                                 |
+| Intel Celeron N4020 | 50 - 200 ms     | Inference speeds on CPU were ~ 800 ms, greatly depends on other loads |
+| Intel NCS2 VPU      | 60 - 65 ms      | May vary based on host device                                         |
```

Comment:
```suggestion
| Intel NCS2 VPU      | 60 - 65 ms      | May vary based on host device                                         |
| Intel i5 1135G7     | 10 - 15 ms      |                                                                                           |
```

Here's the number from my laptop.  I just used the number from the detector inference speed, while running a stock test video of some pedestrians in a street. It's a fairly busy scene, but only one source.  Not sure how to make these numbers comparable or what notes you might want to add to this.

Reply:
That's good because I used the exact same test video for testing the Celeron N4020 so results should line up. I agree it is difficult to standardize, I saw values around 200 ms and then after some time it started dropping and stayed around 30 ms. I think there are already a lot of disclaimers around the values, just helps to give people somewhat of an idea what to expect.